components/table/ActionColumn.razor

```diff
@@ -26,13 +31,42 @@ else if (IsColGroup)
 }
 else if (IsHeader && HeaderColSpan != 0)
 {
+    @if (AppendExpandColumn)
+    {
+        <th class="ant-table-cell ant-table-row-expand-icon-cell"></th>
+    }
+
     <th class="@ClassMapper.Class" style="@FixedStyle @HeaderStyle" colspan="@HeaderColSpan">
         @if (TitleTemplate != null)@TitleTemplate else @Title
     </th>
 }
 else if (!IsHeader && RowSpan != 0 && ColSpan != 0)
 {
+    @if (AppendExpandColumn)
```

Comment:
You're bringing other commits to this PR.

Reply:
Crap... I was doing this on master, than I realized this is should be feature... I will redo this PR from feature branch.